Poker.BE/Poker.BE.Domain/Game/Turn.cs

```diff
@@ -8,7 +8,52 @@ namespace Poker.BE.Domain.Game
 {
     public class Turn
     {
-        // TODO: complete - set team member to do this
+        #region Fields
+        private Player currentPlayer;
 
+        public Player CurrentPlayer { get { return currentPlayer; } set { currentPlayer = value; } }
+        #endregion
+
+        #region Constructors
+        public Turn(Player player)
+        {
+            this.CurrentPlayer = player;
+        }
+        #endregion
+
+        #region Methods
+        public void Check()
+        {
+            ///  Do noting? 
```

Comment:
this is a stub? please use `///<summary>...` statement for documentation (of classes, namespaces, methods, properties... etc.) not for comments. when you want to comment something - use `/* ... */` or `//`. it is best to put `// NOTE: ` or ` // TODO:` so you later can find this comment easily.